ChattoAdditions/Source/Input/Photos/PhotosInputView.swift

```diff
@@ -38,12 +38,31 @@ public protocol PhotosInputViewProtocol {
     var presentingController: UIViewController? { get }
 }
 
+public enum PhotosInputViewPhotoSource {
+    case camera
+    case gallery
+}
+
 public protocol PhotosInputViewDelegate: class {
+    @available(*, deprecated, renamed: "inputView(_:didSelectImage:source:)")
     func inputView(_ inputView: PhotosInputViewProtocol, didSelectImage image: UIImage)
+    /// If you implement this method, deprecated one will not be called
```

Comment:
Let's not keep the deprecated function, in this case it would be much easier to resolve a compilation issue during version update then to support both implementations